site/components/src/main/java/uk/nhs/digital/intranet/components/AnnouncementComponent.java

```diff
@@ -0,0 +1,13 @@
+package uk.nhs.digital.intranet.components;
+
+import org.hippoecm.hst.core.component.HstComponentException;
+import org.hippoecm.hst.core.component.HstRequest;
+import org.hippoecm.hst.core.component.HstResponse;
+import uk.nhs.digital.common.components.BaseGaContentComponent;
+
+public class AnnouncementComponent extends BaseGaContentComponent {
```

Comment:
Can you directly use BaseGaContentComponent in the hst:page? it removes the extra component being instantiated

Reply:
Oooh, that's what you meant. It shouldn't be any problem to re-use the same Java component for multiple hst:pages, correct?

Reply:
On the other hand, I would argue against this on the basis that this is not something done in the codebase (or at least I didn't notice it). My view on this is that every page should have its own component, and that the `BaseGaContentComponent` should only be extended, not directly used. The same as with `BaseHstDocument`.

Reply:
I believe it is recommended to reuse these base components for multiple simple doctypes, but its not a big deal either way, I'll approve.